client/src/modules/list/components/ArchivedList.jsx

```diff
@@ -33,6 +34,7 @@ class ArchivedList extends PureComponent {
 }
 
 ArchivedList.propTypes = {
+  isOwner: PropTypes.bool,
```

Comment:
shouldn't be required?

Reply:
Boolean values don't have to be required if you don't pass them they will be falsy